docs/Team.adoc

```diff
@@ -5,6 +5,8 @@
 
 We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
 
+Other repositories such as https://github.com/se-edu/collate[Collate] and https://github.com/se-edu/learningresources[Learning Resources] manages their own list of contributors.
```

Comment:
Feels a bit weird to have this injected in the middle of nowhere, maybe have a section for it?

```adoc
== Current Team (non-addressbook)
Repositories such as ...

== Current Team (addressbook)
...
```

Reply:
Yup, looks better!!

Reply:
@yamgent I put the `Current Team (non-addressbook)` segment after the `Current Team (addressbook)` instead, because it seems more intuitive to put the major segment before the minor segment.